src/stats/stat.js

```diff
@@ -1,25 +1,38 @@
 'use strict'
 
-const EventEmitter = require('events')
-const Big = require('bignumber.js')
+const { EventEmitter } = require('events')
+const Big = require('bignumber.js').default
 const MovingAverage = require('moving-average')
 
 class Stats extends EventEmitter {
+  /**
+   *
+   * @param {string[]} initialCounters
+   * @param {Object} options
+   * @param {boolean} options.enabled
+   * @param {number} options.computeThrottleTimeout
+   * @param {number} options.computeThrottleMaxQueueSize
+   * @param {import('.').AvarageIntervals} options.movingAverageIntervals
```

Comment:
```suggestion
   * @param {import('.').AverageIntervals} options.movingAverageIntervals
```

please use full path and extract type to types.ts